src/pycatcher/diagnostics.py

```diff
@@ -454,5 +457,9 @@ def build_stl_outliers_plot(df) -> plt:
     for date in df_outliers.index:
         plt.axvline(datetime(date.year, date.month, date.day), color='k', linestyle='--', alpha=0.5)
         plt.scatter(df_outliers.index, df_outliers.iloc[:, -1], color='r', marker='D')
-    else:
+
+    # If the datetime index is not unique, print a warning
```

Comment:
Broke this up due to the error: _useless-else-on-loop / Else clause on loop without a break statement, remove the else and de-indent all the code inside it (col 4)_